# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: image/FrameAnimator.h
# Commit: 1d9384906287
# Full Hash: 1d93849062870a8d492eeef2150ecea174cb6223
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2019-05-26 21:12:53
# Regressor Bug: 1550422
# File Overlap Count: 1
# Description:
#   Bug 1550422 - P15. Move Skip and Once gfxPrefs to StaticPrefs. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D31259
# ==============================================================================

diff -r f5ebdcca640d -r 1d9384906287 image/FrameAnimator.h
--- a/image/FrameAnimator.h	Sun May 26 14:29:56 2019 +0000
+++ b/image/FrameAnimator.h	Sun May 26 14:30:14 2019 +0000
@@ -8,13 +8,13 @@
 #define mozilla_image_FrameAnimator_h
 
 #include "mozilla/Maybe.h"
+#include "mozilla/StaticPrefs.h"
 #include "mozilla/TimeStamp.h"
 #include "gfxTypes.h"
 #include "imgFrame.h"
 #include "nsCOMPtr.h"
 #include "nsRect.h"
 #include "SurfaceCache.h"
-#include "gfxPrefs.h"
 
 namespace mozilla {
 namespace image {
@@ -80,7 +80,7 @@
    * Sets the composited frame as valid or invalid.
    */
   void SetCompositedFrameInvalid(bool aInvalid) {
-    MOZ_ASSERT(!aInvalid || gfxPrefs::ImageMemAnimatedDiscardable());
+    MOZ_ASSERT(!aInvalid || StaticPrefs::ImageMemAnimatedDiscardable());
     mCompositedFrameInvalid = aInvalid;
   }
 